src/js/parsers/section.js

```diff
@@ -6,10 +6,17 @@ import {
   VALID_MARKUP_SECTION_TAGNAMES
 } from 'content-kit-editor/models/markup-section';
 
+function popMany(arr, count) {
```

Comment:
:+1: it's actually `array.splice(-length, +length)` (no `[]`), but yeah. Changing this. I did that because I can never remember the arguments to splice